### PR TITLE
feat: Update to WindowsAppSDK 1.5

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.Windows.csproj [hunk 1]
```diff
@@ -54,7 +54,7 @@
 		<PackageReference Include="Uno.Core.Extensions.Disposables" />
 		<PackageReference Include="Uno.Core.Extensions.Compatibility" />
 		<PackageReference Include="Uno.Core.Extensions" />
-		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230913002" />
+		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240227000" />
 		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
```
Comment (Youssef1313) [inline src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.Windows.csproj:58]:
```suggestion
		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
```

File: src/SamplesApp/SamplesApp.Windows/SamplesApp.Windows.csproj [hunk 2]
```diff
@@ -43,7 +43,7 @@
 			<Version>4.47.0</Version>
 		</PackageReference>
 
-		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230913002" />
+		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240227000" />
 		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
```
Comment (Youssef1313) [inline src/SamplesApp/SamplesApp.Windows/SamplesApp.Windows.csproj:47]:
```suggestion
		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
```

**Review by MartinZikmund — APPROVED** [review]

**Review by dansiegel — COMMENTED** [review]
If we're going to be updating the Uno API to match WindowsAppSDK 1.5.. then we should also be updating the NuGet reference for WindowsAppSDK to 1.5. IMHO it makes no sense to align to an API version that is newer than the version you get by default when installing Uno.WinUI.

**Comment by Youssef1313** [comment]
> If we're going to be updating the Uno API to match WindowsAppSDK 1.5.. then we should also be updating the NuGet reference for WindowsAppSDK to 1.5. IMHO it makes no sense to align to an API version that is newer than the version you get by default when installing Uno.WinUI.

Will do in a separate PR as it needs an update to `Uno.UI.Toolkit.Windows.csproj` as well.

**Comment by jeromelaban** [comment]
> Will do in a separate PR as it needs an update to Uno.UI.Toolkit.Windows.csproj as well.
 @dansiegel @Youssef1313 it's not needed to update `Uno.UI.Toolkit.Windows.csproj`, as it forces the developers to use 1.5 when they upgrade.

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**